test/tests/expected/global_deps_slim.js

```diff
@@ -0,0 +1,8 @@
+[
+    global,
+    function (stealRequire, stealExports) {
+        stealRequire(1);
+        stealRequire(2);
+        var GLOBAL = 'I don\'t like "Quotes"';
```

Comment:
I don't totally understand... how do we currently know what a global exports of they don't provide through metadata? Steal guesses at runtime by looking at the `window` before and after. We don't want to do that in slim so I think configuration is the only way, right?